x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts

```diff
@@ -179,18 +203,65 @@ export type Name = t.TypeOf<typeof name>;
 export const nameOrUndefined = t.union([name, t.undefined]);
 export type NameOrUndefined = t.TypeOf<typeof nameOrUndefined>;
 
+export const operator = t.keyof({
+  equals: null,
```

Comment:
You can use maybe: `OperatorEnum['EQUALS']: null` here. That would reduce one area of manual typing.